contracts/lib/Structs.sol

```diff
@@ -26,44 +28,43 @@ library Structs {
     }
 
     struct Block {
-        uint256 proposerId;
-        uint256[] ids;
-        uint256[] medians;
+        uint32 proposerId;
+        uint32[] medians;
         uint256 iteration;
         uint256 biggestInfluence;
-        bool valid;
     }
 
     struct Dispute {
+        uint8 assetId;
+        uint32 lastVisitedStaker;
         uint256 accWeight;
-        uint256 median;
-        uint256 lastVisited;
-        uint256 assetId;
+        uint256 accProd;
+        // uint32 median;
     }
 
     struct Job {
-        uint256 id;
-        uint256 epoch;
-        string url;
-        string selector;
-        string name;
-        bool repeat;
         bool active;
+        bool repeat;
+        uint8 id;
+        uint8 assetType;
         address creator;
-        uint256 result;
-        uint256 assetType;
+        uint32 epoch;
+        uint32 result;
```

Comment:
I dont think so result is required anymore with fulfillAsset not present. this will always be zero.

Reply:
removed. but delegator is now broken. should create new issue to fix delegator and point it to block manager